src/view/components/Drawer/DrawerContents/index.tsx

```diff
@@ -1,22 +1,31 @@
 import * as React from 'react';
 import { Content, List, Button, Text } from 'native-base';
-import { routes, RouteInfo, Route } from '../../../navigation/navigator';
+
```

Comment:
empty line here hehe do we need to be separating imports?

Reply:
I think we should be separating local imports from packages, but tbh the only reason I was doing it was because I thought there was a linting rule for it :P